tests/main/views/test_services.py

```diff
@@ -1404,6 +1404,31 @@ def test_supplier_name_in_service_data_is_shadowed(self):
             assert response.status_code == 200
             assert data['services']['supplierName'] == u'Supplier 1'
 
+    @mock.patch('app.search_api_client')
+    def test_cannot_update_existing_service_by_put(self, search_api_client):
+        with self.app.app_context():
+            search_api_client.return_value = "bar"
+            response = self.client.put(
```

Comment:
Probably not worth assigning to `response` here as the value is never used?  (I have a feeling @benvand's Flake8 stuff will complain about this.)